customer_data_platform/authorized_api/customer_data_platform_log_api.yaml

```diff
@@ -0,0 +1,157 @@
+servers:
+  - url: https://organization.piwik.pro
+paths:
+  /api/cdp/log/v1/app/{app_id}/audience/{audience_id}/version/{audience_version}/profile/count:
+    get:
+      summary: Count profiles in audience
+      security:
+        - JWTAuth: []
+      operationId: get-log-audience-version-profile-count
+      description: Audience's profiles count
+      parameters:
+        - $ref: "../components.yaml#/components/parameters/AppId"
+        - in: path
+          name: audience_id
+          description: An audience UUID for which profiles count is requested
+          schema:
+            type: string
+            format: uuid
+            example: 81217cfd-c29e-4c7b-9310-936598339971
+          required: true
+        - in: path
+          name: audience_version
+          description: An audience version for which we are counting profiles
+          schema:
+            type: integer
+            minimum: 0
+            example: 2
+          required: true
+      responses:
+        200:
+          content:
+            application/vnd.api+json:
+              schema:
+                type: integer
+                minimum: 0
+                example: 14
+          description: Number of profiles in given audience version
+        400:
+          $ref: "../components.yaml#/components/responses/BadRequestError"
+        401:
+          $ref: "../components.yaml#/components/responses/UnauthorizedError"
+        402:
+          $ref: "../components.yaml#/components/responses/ModuleDisabledError"
+        404:
+          $ref: "../components.yaml#/components/responses/NotFoundError"
+        422:
+          $ref: "../components.yaml#/components/responses/UnprocessableEntityError"
+        500:
+          $ref: "../components.yaml#/components/responses/InternalServerError"
+  /api/cdp/log/v1/app/{app_id}/audience/{audience_id}/version/{audience_version}/profile/last:
+    get:
+      summary: List last profiles belonging to given audience
+      security:
+        - JWTAuth: []
+      operationId: get-log-audience-version-profile-last
+      description: Returns at most 1000 items
+      parameters:
+        - $ref: "../components.yaml#/components/parameters/AppId"
+        - in: path
+          name: audience_id
+          description: An audience UUID for which profiles are requested
+          schema:
+            type: string
+            format: uuid
+            example: ca5b925e-6888-46fe-a0bc-55907c5af050
+          required: true
+        - in: path
+          name: audience_version
+          description: An audience version for which profiles belong
+          schema:
+            type: integer
+            minimum: 0
+            example: 2
+          required: true
+        - in: query
+          name: limit
+          description: How many last profiles should be returned
+          schema:
+            type: integer
+            maximum: 1000
+            minimum: 1
+            example: 10
```

Comment:
Can we say that the default is 100? Same for offest, it should be 0 by default.

Reply:
Sure!